Randomizer.py

```diff
@@ -0,0 +1,24 @@
+import random
+
+
+def randomly(list_of_students):
+    student_dict_reverted = {}
+    for elem in list_of_students:
+        student_dict_reverted[elem] = 0
+    print('Do you want to make your choice?', '\n', 'Press y to do it, n to exit')

```

Comment:
Maybe ask whether we want to peek one more student at the end of while loop. It's a strange situation when we run our script and then entered `n`.

Reply:
Students should be selected constantly, I see no reason to make a confirmation after each choice.

Reply:
However I have to confirm that I want to make a new choice 
```
python3 test.py 
Do you want to make your choice? 
 Press y to do it, n to exit
y
D
y
B
y
B
y
B
```

Reply:
I suggest to consider something like this:
```
D
Proceed? (y/n) y
B
Proceed? (y/n)  y
B
Proceed? (y/n) y
B
```